<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QWidget" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>530</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,0,0">
     <item>
      <widget class="QLineEdit" name="leSearch">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Поиск...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSort">
       <property name="placeholderText">
        <string>Сортировать...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClientAccount">
       <property name="text">
        <string>Аккаунт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOpenCart">
       <property name="text">
        <string>Корзина</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Товары</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tvProducts"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
